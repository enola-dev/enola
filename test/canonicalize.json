{
  "numbers": [333333333.33333329, 1E30, 4.50,
    2e-3, 0.000000000000000000000000001],
  "string": "\u20ac$\u000F\u000aA'\u0042\u0022\u005c\\\"\/",
  "literals": [null, true, false],

  "\u20ac": "Euro Sign",
  "\r": "Carriage Return",
  "\ufb33": "Hebrew Letter Dalet With Dagesh",
  "1": "One",
  "\ud83d\ude00": "Emoji: Grinning Face",
  "\u0080": "Control",
  "\u00f6": "Latin Small Letter O With Diaeresis"
}
