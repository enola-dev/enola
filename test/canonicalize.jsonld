[
  {
    "@id": "http://example.enola.dev/Picasso",
    "https://schema.org/name": [
      {
        "@value": "Pablo Picasso"
      }
    ]
  },
  {
    "https://schema.org/name": [
      {
        "@value": "Salvador Domingo Felipe Jacinto Dalí"
      }
    ],
    "@id": "http://example.enola.dev/Dalí"
  }
]
