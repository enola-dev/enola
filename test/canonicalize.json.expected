{"\r":"Carriage Return","1":"One","literals":[null,true,false],"numbers":[3.333333333333333E8,1.0E30,4.5,0.002,1.0E-27],"string":"â‚¬$\u000f\nA'B\"\\\\\"/","Â€":"Control","Ã¶":"Latin Small Letter O With Diaeresis","â‚¬":"Euro Sign","ðŸ˜€":"Emoji: Grinning Face","ï¬³":"Hebrew Letter Dalet With Dagesh"}